Pinta.Tools/Tools/TextTool.cs

```diff
@@ -156,14 +156,22 @@ protected override void OnBuildToolBar (Gtk.Box tb)
 		tb.Append (font_label);
 
 		if (font_button == null) {
-			font_button = new Gtk.FontButton {
+			var fontDialog = new Gtk.FontDialog {
```

Comment:
Minor suggestion for coding style - we prefer using `Gtk.FontDialog fontDialog = new { ... };`